status.js

```diff
@@ -1,36 +1,34 @@
-function init() {
-  return new Promise(resolve => {
-    window.onload = () => {
-      var tbody = document.getElementById("channel-status");
-      tbody.innerHTML = "<tr><td class='jsfallback' colspan='5'>Loading data from Prometheus...</td></tr>";
-      resolve();
-    };
-  });
-}
+var init = new Promise(resolve => {
+  window.onload = () => {
+    var tbody = document.getElementById("channel-status");
+    tbody.innerHTML = "<tr><td class='jsfallback' colspan='5'>Loading data from Prometheus...</td></tr>";
+    resolve();
+  }
+});
```

Comment:
Considering it's only used once, we could move the whole `new Promise(...)` in-place of where `init` is used.